### DISCUSSION TITLE
need some info

### DISCUSSION THREAD

**Discussion by malikshi** [helmiau/HelmiWrt-OS#26] [General]
this firmware needs to work along with your other repo [openwrt config](https://github.com/helmiau/openwrt-config) right? currently, I am using other firmware but haven't been tweaking for Libernet nor Xderm. can you add [xderm](https://github.com/ryanfauzi1/xderm-mini_GUI) as another option for Libernet?

**Comment by malikshi** [comment]
Your firewall setting really weird.i am add new interface with correct setting, it's connected but no internet access/ still using original isp not via openvpn/l2tp/wireguard. With same setting the other fw work well no issue like that

**Comment by helmiau** [comment]
> this firmware needs to work along with your other repo [openwrt config](https://github.com/helmiau/openwrt-config) right? currently, I am using other firmware but haven't been tweaking for Libernet nor Xderm. can you add [xderm](https://github.com/ryanfauzi1/xderm-mini_GUI) as another option for Libernet?

not at all, i'm only using SSR+/OpenClash/Passwall

**Comment by helmiau** [comment]
> Your firewall setting really weird.i am add new interface with correct setting, it's connected but no internet access/ still using original isp not via openvpn/l2tp/wireguard. With same setting the other fw work well no issue like that

Then you need to reconfigure your firewall instead of mine, because my fw build is built for my daily usage for Load Balancing with 3 ISP USB Modem HiLink

**Comment by malikshi** [comment]
Can you recommend me good usb modem? Currently I'm using B818 orbitmax

**Comment by helmiau** [comment]
> Can you recommend me good usb modem? Currently I'm using B818 orbitmax

e3372h-607
what connection you use for orbitmax? Usb or LAN?

**Comment by malikshi** [comment]
direct LAN to rpi4 LAN, for EU 300usb to lan used for AP router

**Comment by helmiau** [comment]
> direct LAN to rpi4 LAN, for EU 300usb to lan used for AP router

i think yours will work if you use external USB to Ethernet hub (as ISP)
so the Pi 4 LAN port will be used for connection to AP

**Comment by helmiau** [comment]
Moved this issues to discussions page
[here for the discussions link](https://github.com/helmiau/OpenWrt-Rpi/discussions/26)

**Comment by malikshi** [comment]
Rofl. It's not how you're move issues into discussion
